meinberlin/apps/budgeting/phases.py

```diff
@@ -13,8 +13,7 @@ class RequestPhase(phases.PhaseContent):
     view = views.ProposalListView
 
     name = _('Request phase')
-    description = _('Request budgeting and get feedback through rates and '
-                    'comments.')
+    description = _('Participants can submit ideas, comment on and rate them.')
```

Comment:
Are these the new descriptions? 
Maybe in English we should stick to proposal instead of idea, so it might be less confusing?

Reply:
Yes, they are in the story.. @CarolingerSeilchenspringer ok to change ideas to proposasl?